lib/package.js

```diff
@@ -164,6 +164,13 @@
           skipExtension = true;
       });
 
+    // exact global meta or meta with any content after the last wildcard skips extension
+    if (!skipExtension && System.modules)
+      getMetaMatches(System.modules, pkgName, subPath, function(metaPattern, matchMeta, matchDepth) {
```

Comment:
`getMetaMatches` is designed for package meta, so won't work here I don't think.